### ISSUE TITLE
Time of day setting ?

### ISSUE THREAD

**Issue by griemens** [walkinside/sdk#30]
Will there be an alternative for the time of day setting in the skybox for future versions ?
I noticed VRSkybox.Time of day has been flagged as obsolete. Is there an alternative I can use ?

Thanks,

**Comment by kveretennicov** [comment]
Do you use this setting already in a plug-in or do you plan to use it? If yes - to what end?

**Comment by griemens** [comment]
Currently I am developing a plugin to improve user experience to prepare a model for presentations and easily restore views (Kind of Walkinside viewpoints on steroids).
MyViewpoint will have the following information to restore besides camera info.

-          Terrain on/off (in some cases you like to have a view where terrain is blocking the view, foundations of building for example. In some views terrain is required for walking)

-          Ocean on/off (Same as terrain issue, for marine models)

-          Time of day (so it is exactly when view point was saved)

-          Clipping

-          Avatar Distance + 3rth person mode + Size (so the 3d image is exactly as displayed)

-          Branches switched off/on (in case of isolation use case)

-          Render mode

-          Redline groups items enabled/disabled

-          Static skybox texture

-          Gravity, and collision

-          Walking Speed (when MyViewPoint is set far away of model and model is huge)

-          …

Started with time of day cause I also made a plugin for me to set my preferred default settings on models in one click.

è Static skybox -> avoids constant fan usage

è Static ocean -> avoids constant fan usage

è Time of day

Note: Time of day can be set through SDK when static skybox Is enabled through. Walkinside does not allow that. It is required to be dynamic skybox.

Thank you.

**Comment by kveretennicov** [comment]
Interesting, so it's like a complete snapshot of Viewer's session state and settings.

CC @walkinside/devs @walkinside/staff

**Comment by griemens** [comment]
Yes that’s the would be the ideal. But the 3D window size and such, that will not be possible I believe. For example the screen resolutions between machines could be different.
